src/runtime/cpu/kernels/kernels.cpp

```diff
@@ -1,15 +1,21 @@
 #include "runtime/cpu/kernels/kernels.hpp"
 
+// FIXME: It can something general like:
```

Comment:
@KSTconsult Just put a reminder, it future it can be decomposed to `Mul` and `Add` operations

Reply:
Decomposed in this PR: https://github.com/NeuralDeepWorks/deepworks/pull/23